### PR TITLE
fix: support alpine linux by retrying grep with -r

### LINKED ISSUE #46: grep -R fails on alpine linux
See this commit [here](https://github.com/nros/chrome-finder/commit/0afd72ee5cd2f294a0e003c25c473995db6bf80e). chrome-finder should fall back to using lowercase R on platforms that don't support uppercase R.

### PR DESCRIPTION
**Pull request by AkshayIyer12**
Chrome-finder will fall back to using lowercase R on platforms that don't support uppercase R.

fixes #46

### CODE REVIEW

**Comment by patrickhulce** [comment]
unfortunately this is a rather esoteric bug that we won't really have a great way to unit test, probably resort to manually testing to make sure things seem ok

@danbopes do you have alpine linux readily available to you such that you could give this a quick spin?

File: chrome-finder.ts [hunk 1]
```diff
@@ -229,10 +229,17 @@ function findChromeExecutables(folder: string): Array<string> {
...
     //    /opt/google/chrome/google-chrome --profile-directory
     //    /home/user/Downloads/chrome-linux/chrome-wrapper %U
-    let execPaths = execSync(`grep -ER "${chromeExecRegex}" ${folder} | awk -F '=' '{print $2}'`)
-                        .toString()
-                        .split(newLineRegex)
-                        .map((execPath: string) => execPath.replace(argumentsRegex, '$1'));
+    let execPaths;
+
+    try {
+      execPaths = execSync(`grep -ER "${chromeExecRegex}" ${folder} | awk -F '=' '{print $2}'`);
```
Comment (patrickhulce) [inline chrome-finder.ts]: want to add a quick comment about some platforms not supporting `-R`, maybe with a link to the issue?

**Comment by danbopes** [comment]
@patrickhulce I've gone ahead and added some docker tests [here](https://github.com/AkshayIyer12/chrome-launcher/pull/1).

**Comment by paulirish** [comment]
thx very much for contributing this, @AkshayIyer12. I must merged in #61 which includes your work here.

**Comment by AkshayIyer12** [comment]
Thanks @patrickhulce :+1:

**Pull request closed without merging**